packages/manager/apps/telecom/src/app/telecom/telephony/svaWallet/kyc/identity/overview-identity/overview-identity-edit/confirm/confirm.controller.js

```diff
@@ -0,0 +1,24 @@
+export default class ConfirmController {
+  /* @ngInject */
+  constructor($uibModalInstance, $translate) {
+    this.$uibModalInstance = $uibModalInstance;
+    this.$translate = $translate;
+  }
+
+  $onInit() {
+    this.confirmationCode = this.$translate.instant(
+      'telephony_billingAccount_svaWallet_kyc_identity_confirm_code',
+    );
+    this.confirmationPattern = new RegExp(`^${this.confirmationCode}$`);
+  }
+
+  confirm() {
+    if (!this.form.$invalid) {
+      this.$uibModalInstance.close();
+    }
```

Comment:
Same here, you should avoid click with no action if `this.form$invalid` is truthy. You might need to use `primary-disabled`/`primary-action` instead than a form.